### ISSUE TITLE
Pass & Fail - how about adding "Not applicable" to (manual) checks that can not be checked

### ISSUE THREAD

**Issue by BogdanCerovac** [microsoft/accessibility-insights-web#3228]
## Is your feature request related to a problem? Please describe.
I think we must think of some checks in terms of trinary logic as we should be able to present the results with "Pass" / "Fail" / "Not applicable". Current solution for the final report now requires some additional manual editing for cases that could be marked as "Not applicable" (or some similar naming suggesting that it is nor pass, nor fail, but not checked).

## Describe the desired outcome
For example: I do manual tests on a page that does not include any audio or video elements. Automatic test detect that they are not present and suggest "Not applicable" for all Multimedia checks by default. But I can still manually override it as it may happen that I inject some video dynamically for example, that can potentially not be detected with automatic checking alone.

## Describe alternatives you've considered
As described in the desired outcome:
- test cases for elements not present on the page get "Not applicable" by default, but this can be manually changed by the user.
- all test cases for all elements that can potentially be "Not applicable" should have a an option to be "Not applicable".
- all test cases that must be either Pass or Fail should not have an option for "Not applicable" (for example cases that must always be checked - like for example correct language on the document level, alt image text meaning corresponding to image in the context etc.).

## Additional context
Possibility to then group end report by "Not applicable" too, and maybe an option to hide all "Non applicable" if it seems reasonable.

**Comment by ferBonnin** [comment]
Thanks for your feature request @BogdanCerovac!
This suggestion is something we considered back when we were creating Assessment in Accessibility Insights. To simplify the user workflow and reduce the complexity, we decided to have only 3 states inside Assessment: Pass, Fail and Incomplete; with Pass and Fail the only ones the users need to add. 

After reviewing this with the team, we have decided to keep just this 3 states but we understand that there is a need to add comments sometimes so we have feature request #485 open for comments; it will allow users to add a comment to the list of instances; a similar option could be considered for manual requirements in Assessment if that could be helpful.
